frontend/doc-site/11_RoadMap.mdx

```diff
@@ -21,140 +21,181 @@ Our technology roadmap aims to:
 
 We are planning to develop the following features on the Data Portal application. The Status column has one of three statuses:
 
-* Development: Our engineering team is currently working hard to deliver this feature.
-* Discovery: We're in the process of defining requirements and prototyping this feature.
-* Not Started: Self explanatory! We haven't started Discovery yet.
+- Development: Our engineering team is currently working hard to deliver this feature.
+- Discovery: We're in the process of defining requirements and prototyping this feature.
+- Not Started: Self explanatory! We haven't started Discovery yet.
 
 <table>
   <thead>
     <tr>
-      <th style="text-align:left">
+      <th style={{ textAlign: "left" }}>
```

Comment:
since this is technically JSX and not HTML, style attribute values should be provided as an object not a string. https://reactjs.org/docs/dom-elements.html